client-network/src/network_ops/default_network_ops_client.rs

```diff
@@ -472,8 +445,52 @@ where
     }
 
     #[inline]
-    fn get_staked_state(&self, address: &StakedStateAddress) -> Result<StakedState> {
-        self.get_staked_state_account(address)
+    fn get_staked_state(
+        &self,
+        name: &str,
+        address: &StakedStateAddress,
+        verify: bool,
+    ) -> Result<StakedState> {
+        let mstaking = if verify {
```

Comment:
instead of mstaking,
how about full word variable name ?

Reply:
I believe the name style of `mxxx` is adopted from Haskell, because I've seen this style in our code base several times, so I keep on using it. Basically in these cases `mxxx` means `optional_xxx`.